services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -684,6 +696,7 @@ private void doBatchInstall(final CommandMessage message) throws FlowCommandExce
 
     private void doPortsCommandDataRequest(CommandMessage message) {
         try {
+            logger.info("Getting port data");
```

Comment:
switch id

Reply:
As I understand from code, here we are iterating through all switches. So there is no much useful information here. Except the fact of getting ports data.